pkg/util/command_util.go

```diff
@@ -442,8 +442,8 @@ func LookupUser(userStr string) (*user.User, error) {
 	userObj, err := user.Lookup(userStr)
 	if err != nil {
 		unknownUserErr := new(user.UnknownUserError)
-		// only return if it's not an unknown user error
-		if !errors.As(err, unknownUserErr) {
+		// only return if it's not an unknown user error or the passwd does not exist
+		if !errors.As(err, unknownUserErr) && !strings.Contains(err.Error(), "no such file") {
```

Comment:
Instead of matching the error text string, it looks like you can check [`os.IsNotExist(err)`](https://pkg.go.dev/os#IsNotExist)

Reply:
Thanks @imjasonh. Works like a charm. Updated the PR.